docs/reference/release-notes/8.0.0.asciidoc

```diff
@@ -0,0 +1,699 @@
+* Check that `total_definition_length` is consistent before starting a deployment {es-pull}80553[#80553]
+* Fail inference processor more consistently on certain error types {es-pull}81475[#81475]
+* Optimize the job stats call to do fewer searches {es-pull}82362[#82362] (issue: {es-issue}82255[#82255])
+
+Mapping::
+* Remove assertions that mappings have one top-level key {es-pull}58779[#58779] (issue: {es-issue}58521[#58521])
+
+Packaging::
+* Suppress illegal access in plugin install {es-pull}41620[#41620] (issue: {es-issue}41478[#41478])
+
+Recovery::
+* Make shard started response handling only return after the cluster state update completes {es-pull}82790[#82790] (issue: {es-issue}81628[#81628])
+
+SQL::
+* Introduce dedicated node for `HAVING` declaration {es-pull}71279[#71279] (issue: {es-issue}69758[#69758])
+* Make `RestSqlQueryAction` thread-safe {es-pull}69901[#69901]
+
+Search::
+* Check for negative `from` values in search request body {es-pull}54953[#54953] (issue: {es-issue}54897[#54897])
+* Fix `VectorsFeatureSetUsage` serialization in BWC mode {es-pull}55399[#55399] (issue: {es-issue}55378[#55378])
+* Handle total hits equal to `track_total_hits` {es-pull}37907[#37907] (issue: {es-issue}37897[#37897])
+* Improve error msg for CCS request on node without remote cluster role {es-pull}60351[#60351] (issue: {es-issue}59683[#59683])
+* Remove unsafe assertion in wildcard field {es-pull}78966[#78966]
+* Reject zero-length vectors when using cosine similarity {es-pull}82241[#82241] (issue: {es-issue}81167[#81167])
+
+Security::
+* Allow access to restricted system indices for reserved system roles {es-pull}76845[#76845]
+* Auto-generated TLS files under fixed config path {es-pull}81547[#81547] (issue: {es-issue}81057[#81057])
+* Bind to non-localhost for transport in some cases {es-pull}82973[#82973]
+* Correct file ownership on node reconfiguration {es-pull}82789[#82789] (issue: {es-issue}80990[#80990])
+* Display security auto-configuration with fancy unicode {es-pull}82740[#82740] (issue: {es-issue}82364[#82364])
+
+Snapshot/Restore::
+* Fix `GET /_snapshot/_all/_all` if there are no repos {es-pull}43558[#43558] (issue: {es-issue}43547[#43547])
+* Don't fill stack traces in `SnapshotShardFailure` {es-pull}80009[#80009] (issue: {es-issue}79718[#79718])
+* Remove custom metadata if there is nothing to restore {es-pull}81373[#81373] (issues: {es-issue}81247[#81247], {es-issue}82019[#82019])
+
+[[regression-8.0.0]]
+[float]
+=== Regressions
+
+Search::
+* Disable numeric sort optimization conditionally {es-pull}78103[#78103]
+
+[[upgrade-8.0.0]]
+[float]
+=== Upgrades
+
+Infra/Logging::
+* Upgrade ECS logging layout to latest version {es-pull}80500[#80500]
+
+Lucene::
+* Upgrade to Lucene 8.9.0 {es-pull}74729[#74729]
+
+Search::
+* Update Lucene 9 snapshot {es-pull}79701[#79701] {es-pull}79138[#79138] {es-pull}78548[#78548] {es-pull}78286[#78286] {es-pull}73324[#73324] {es-pull}79461[#79461]
```

Comment:
I think these should be removed, as they are superseded by the one below.  The one below should be changed to simply "Upgrade to Lucene 9".

Reply:
Thanks for catching this. I'll leave in the Lucene 9 item and remove the other two.